config/master.d/50-master.conf

```diff
@@ -21,6 +21,7 @@ file_roots:
 pillar_roots:
   base:
     - /usr/share/salt/kubernetes/pillar
+    - /srv/pillar
```

Comment:
We should probably avoid hijacking `/srv/pillar` from the public cloud guys - I think it would be better to use `/etc/caasp/pillar-customizations/` (both inside, and outside the container..)

Reply:
I think they are using that for the very same reason:
https://github.com/kubic-project/caasp-container-manifests/blob/d6b19f7b8bc6f8e57ac4099d35bb3382669afc99/manifests/public.yaml#L169-L171

Which is backed by:
https://github.com/kubic-project/caasp-container-manifests/blob/d6b19f7b8bc6f8e57ac4099d35bb3382669afc99/manifests/public.yaml#L558-L560

So we already have this directory mounted in. I don't think they have anything against renaming `public-cloud-config` to something more generic as `user-pillars`.

@bear454: what do you think about that?

Reply:
You can name the mount whatever you want, really, so long as you don't break import of the pillar file we put there.

That being said, I'm not sure `user-pillars` is the best choice; if someone thinks they have permission to modify our pillar it's going to be a support nightmare. _The public cloud pillar is definitely not intended for user modification._

Reply:
Agree with @bear454  - I think we should keep the public cloud stuff separate to the unsupported "tweaks" an operator can make.

I think making the scope of this too wide is dangerous for supportability reasons, but we've found enough times already that customers are going to make tweaks. And we're going to actively undo those tweaks, making this better - while keeping it as unsupported is IMO the goal :)

Reply:
As a side note - the "prod" env name and generic "/srv/pillar" folder used for public cloud stuff should ideally be renamed to something more specific to avoid confusion :)